src/components/SpeciesComposition/SpeciesComposition.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-card-subtitle v-if="message">{{ message }}</v-card-subtitle>
+  <div v-else>
+    <v-chart ref="chart" class="chart" :option="option" />
+    <download-chart-button filename="species-composition.png" />
+  </div>
+</template>
+
+<script>
+import axios from "axios";
```

Comment:
We should remove axios and use fetch instead. That would be one less dependency. What do you think?

Reply:
Done